Packs/Cisco-umbrella-cloud-security/Integrations/CiscoUmbrellaCloudSecurityv2/CiscoUmbrellaCloudSecurityv2.yml

```diff
@@ -29,27 +29,18 @@ script:
     - name: destination_list_id
       description: The ID of the destination list. Destination lists can be fetched with the `umbrella-destination-lists-list` command.
       required: true
-      isArray: false
```

Comment:
I did not intentionally move all of these `isArray` and `required`. Do you want me to add them back in?